src/api/Nhs.Appointments.Persistance/UserStore.cs

```diff
@@ -21,7 +21,36 @@ public async Task<IEnumerable<RoleAssignment>> GetUserRoleAssignments(string use
         var userDocument = await cosmosStore.GetByIdAsync<UserDocument>(userId);
         return userDocument.RoleAssignments.Select(mapper.Map<RoleAssignment>);
     }
-    
+
+    /// <summary>
+    /// Updates the roles assigned to the user
+    /// </summary>
+    /// <param name="userId"></param>
+    /// <param name="scope"></param>
+    /// <param name="roleAssignments"></param>
+    /// <returns>The original role assignments prior to completion of the update operation</returns>
+    /// <exception cref="Exception"></exception>
+    public async Task<RoleAssignment[]> UpdateUserRoleAssignments(string userId, string scope, IEnumerable<RoleAssignment> roleAssignments)
+    {
+        var originalDocument = await GetOrDefault(userId);
+        if (originalDocument == null)
+        {
+            throw new Exception($"Cannot update role assignments: user '{userId}' not found");
```

Comment:
This behaviour is different to the method it has replaced. This would cause an issue with the current functionality as it will now no longer be possible to assign roles to new users.